stories/Display/TabsStory.tsx

```diff
@@ -31,6 +33,14 @@ const props = {
       description:
         'Sets alignment of Tab. The vertical tabs are changed to horizontal ones for screen size below 768',
     },
+    {
+      name: 'colour',
+      type: 'string',
+      defaultValue: 'black',
+      possibleValue: 'blue, grey',
```

Comment:
There is no `blue` in `ETabColourVariant`

Reply:
But I think we can use `blue` to instead of `grey` because using active color should be more intuitive. Wdyt?